pywr/parameters/_parameters.pyx

```diff
@@ -625,10 +625,14 @@ cdef class MonthlyProfileParameter(Parameter):
     ----------
     values : iterable, array
         The 12 values that represent the monthly profile.
-    lower_bounds : float (default=0.0)
+    lower_bounds : float (default=0.0) or list
```

Comment:
I did a bit more reading about this. These should accept general `array_like` rather than just lists.

```
lower_bounds, upper_bounds : float or array_like (default=0.0)
    Defines the lower and upper bounds when using optimisation. If a float given, same bound applied for every
    month. Otherwise an array like object of length 12 should be given for as separate value each month.
```